conformance/Makefile

```diff
@@ -36,16 +44,26 @@ install-nkg: ## Install NKG with provisioner on configured kind cluster
 	kubectl apply -f ../deploy/manifests/rbac.yaml
 	kubectl apply -f ../deploy/manifests/gatewayclass.yaml
 	kubectl apply -f ../deploy/manifests/service/nodeport.yaml
-	kubectl apply -f provisioner/provisioner.yaml
 
-.PHONY: update-test-kind-config 
-update-test-kind-config: ## Update kind config
-	sed -ir "s|server:.*|server: https://kind-control-plane:6443|" $(KIND_KUBE_CONFIG_FOLDER)/config
+.PHONY: install-nkg-local-build
+install-nkg-local-build: build-and-load-images prepare-nkg-dependencies ## Install NKG from local build with provisioner on configured kind cluster
+	yq '(select(di != 3))' provisioner/provisioner.yaml | kubectl apply -f -
+	yq '(select(.spec.template.spec.containers[].image) | .spec.template.spec.containers[].image="$(NKG_PREFIX):$(NKG_TAG)" | .spec.template.spec.containers[].imagePullPolicy = "Never")' provisioner/provisioner.yaml | kubectl apply -f -
+
+.PHONY: install-nkg-edge
+install-nkg-edge: preload-nginx-container prepare-nkg-dependencies ## Install NKG with provisioner from edge on configured kind cluster
+	kubectl apply -f provisioner/provisioner.yaml
 
 .PHONY: run-conformance-tests
-run-conformance-tests: update-test-kind-config ## Run conformance tests
-	docker run --network=kind --rm -v $(KIND_KUBE_CONFIG_FOLDER):/root/.kube $(PREFIX):$(TAG) \
-		go test -timeout 25m -v . -tags conformance -args --gateway-class=$(GATEWAY_CLASS) --debug --supported-features=$(SUPPORTED_FEATURES) --exempt-features=$(EXEMPT_FEATURES)
+run-conformance-tests: ## Run conformance tests
+	kind load docker-image $(PREFIX):$(TAG)
+	kubectl apply -f tests/conformance_rbac.yaml
+	kubectl run -i conformance --image=$(PREFIX):$(TAG) --image-pull-policy=Never --restart=Never --overrides='{ "spec": { "serviceAccountName": "conformance" }  }' -- go test -v . -tags conformance -args --gateway-class=$(GATEWAY_CLASS) --debug --supported-features=$(SUPPORTED_FEATURES) --exempt-features=$(EXEMPT_FEATURES)
```

Comment:
would it make sense to split this command across multiple lines for readability?